ReactWindows/ReactNative/UIManager/PropertySetter.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using ReactNative.UIManager.Annotations;
+using Newtonsoft.Json.Linq;
```

Comment:
Where in this file is either JArray or JObject used?

---
In reply to: [76174598](https://github.com/ReactWindows/react-native-windows/pull/625#discussion_r76174598) [](ancestors = 76174598)